src/veh_interact.cpp

```diff
@@ -694,6 +701,20 @@ void veh_interact::do_install()
         bool can_install = can_install_part();
 
         const std::string action = main_context.handle_input();
+        if (action == "FILTER" ){
+            filter = string_input_popup( _("Set Filter"),
+                                        50, filter,
+                                        _("{f:} to filter by part flag, {pgt} and {plt} to filter by power, combine with ',', otherwise it searches by name"),
```

Comment:
Eh, I kinda like it, but if you think it's not worth the trouble, then perhaps it should wait until there's some sort of unified way to go about this.